onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -771,3 +950,37 @@ def _dequantize_outputs(self):
             dequantize_node = self._dequantize_value(output.name)
             if dequantize_node is not None:
                 self.new_nodes.append(dequantize_node)
+
+    def calculate_quantization_params(self):
+        if self.tensors_range is None:
+            return None
+
+        self.adjust_tensor_ranges(softmax_0_to_1=False)
```

Comment:
we should keep QDQ and operator oriented to have same behavior. just set the range to 0 to 1 for both.